### PR TITLE
Module 1 Capstone project

### PR DESCRIPTION
**Pull request by oguntayomicheal**
### In this PR, the following tasks were implemented:
- [ ] Add the mobile homepage
- [ ] Add the mobile About page
- [ ] Adapt the mobile home and about pages to desktop versions
- [ ] Create the featured speakers dynamically

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <img src="./assets/airbnb-logo.png" alt="airbnb logo">
+
+        </div>
+        
+    </section>
+    
+    <footer id="footer">
+        <div id="footer_logo"></div>
+        <p>2022 Science fun facts</p>
+    </footer>
```
Comment (Munsa1) [inline about.html]: - [ ] Kindly add the correct color for the footer section, the text is `color: white` and the backround is `color: #272A31`.
- [ ] Also align the footer items correctly as they are aligned in the given design, not to the center but to the left and right. You could use `display: flex` then `justify-content: space between`.
- [ ] Lastly the footer has two blocks of text to the left, Kindly add a second paragraph so as to match the design

#### Your design
image

#### Given design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        </div>
+
+        <div id="companies">
+            <img src="./assets/mozilla-logo.svg" alt="mozilla logo">
+            <img src="./assets/google_logo.png" alt="google logo">
+            <img src="./assets/naver-logo.png" alt="naver logo">
+            <img src="./assets/Daum-Kakao-logo.png" alt="daumkakao logo">
+            <img src="./assets/airbnb-logo.png" alt="airbnb logo">
+
+        </div>
```
Comment (Munsa1) [inline about.html:110]: - [ ] The about page does not have the `Partners` section, kindly remove it.

#### Your design
image

#### Given design
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                    <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+                </nav> 
+                <p class="hello_text">"Hello! great learners"</p>
+                <h1>Science Fun Facts</h1>
+                <p class="brief">The history of science is almost as old as time itself. Science is constantly working 
+                    from the edges of the universe to the innermost cells that make you function
+                </p>
+                <p class="date">All about science</p>
+                <p class="location">Its Latin word, <i>scienta</i> means knowledge.</p>
+            </div>
```
Comment (Munsa1) [inline index.html:52]: - [ ] Kindly  Increase the margin from ther left so as to make your design look similar to the given design
- [ ] Also increase the font size for the h1 tag so as to make it look more similar to the given design

#### Given design
image

#### Your design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+    </div>
+
+    <section class="sci_facts">
+       <div class="title">
+        <h2 class="h2_sci_facts">Some Science facts</h2>
+        <div class="line"></div>
+       </div>
+
+       <div id="desk_facts">
```
Comment (Munsa1) [inline index.html]: - [ ] These articles are five and not six kindly fix this, also they have to be wrapped in an article tag and not the div tag

#### Your design
image

#### Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Mike-devloper, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by jasonfelice — APPROVED** [review]
Hi @Mike-devloper, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

<p align="center"></p>

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ons or comments in the PR thread if something is not 100% clear.

File: about.html (hunk 1 above)
Comment (oguntayomicheal) [reply about.html]: Hi @Munsa1, thanks for the review. All required changes would be implemented according to the reviews given.

**Comment by oguntayomicheal** [comment]
Hi @jasonfelice, thanks for the review and project approval.

**Pull request merged**